llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -562,6 +562,89 @@ def IREELinalgExt_PackOp : IREELinalgExt_Op<"pack", [
   }];
 }
 
+def IREELinalgExt_UnPackOp : IREELinalgExt_Op<"unpack", [
+  DeclareOpInterfaceMethods<ReifyRankedShapedTypeOpInterface>,
+  DeclareOpInterfaceMethods<LinalgExtInterface>,
+  DeclareOpInterfaceMethods<TilingInterface,
+    ["getIterationDomain",
+     "getLoopIteratorTypes",
+     "generateScalarImplementation"]>,
+  DeclareOpInterfaceMethods<MemoryEffectsOpInterface>
+]>{
+  let summary = "unpack operation";  
```

Comment:
Please add more op documentation

Reply:
yes, I saw. Doing that now.

Reply:
Ok, added. If it is good to go, let's, please merge.